build.gradle

```diff
@@ -65,7 +65,7 @@ def ensureBuildPrerequisites(requiredJavaVersion, buildPrerequisitesMessage) {
 }
 ensureBuildPrerequisites(requiredJavaVersion, buildPrerequisitesMessage)
 
-final htsjdkVersion = System.getProperty('htsjdk.version', '2.19.0')
+final htsjdkVersion = System.getProperty('htsjdk.version', '2.20.0')
```

Comment:
didn't you want to upgrade to 2.22?

Reply:
No, 2.20.0 is the right version, I made a mistake in the title of the pr.